store/store.go

```diff
@@ -766,53 +766,77 @@ func (s *SnapUbuntuStoreRepository) SuggestedCurrency() string {
 	return s.suggestedCurrency
 }
 
-// Optionally specify parameters to the store purchase.
-// Currency: ISO 4217 code as string
-// BackendID: The payment backend to use, e.g. "credit_card", "paypal".
-// MethodID: The unique identifier for the payment method, retrieved from the pay server.
-type BuyOptions struct {
-	Currency  string
-	BackendID string
-	MethodID  int64
-}
-
-// When the purchase state is "InProgress" (i.e. requires user interaction to complete)
-// this struct contains the information required to complete the purchase.
-type BuyRedirect struct {
+// BuyParameters specifies parameters for store purchases.
+type BuyParameters struct {
```

Comment:
The original name with Options was a better fit here. The convention is already used elsewhere.